src/App.js

```diff
@@ -5,15 +5,16 @@ import timelineData from './data/timeline.json';
 import Timeline from './components/Timeline';
 
 function App() {
-  console.log(timelineData);
+  // console.log(timelineData);
 
-  // Customize the code below
+  
   return (
     <div className="App">
       <header className="App-header">
-        <h1 className="App-title">Application title</h1>
+        < h1 className = "App-title" > Ada Lovelace's Social Media Feed </h1>
```

Comment:
Just noting you can use `timelineData` here.

```suggestion
        < h1 className = "App-title" > {timelineData.person}'s Social Media Feed </h1>
```